README.md

```diff
@@ -36,6 +36,31 @@ projects. This tool does not only detect them, but also fixes them for you.
 
 ## Documentation
 
+### Run with docker
+
+You can take a ready built docker image to run ``php-cs-fixer``.
+
+```console
+docker run -v $(pwd):/var/www ghcr.io/php-cs-fixer/php-cs-fixer:latest fix src
+```
+
+or integrate as check into gitlab-ci like this
+
+```yaml
+php-cs-fixer:
+  image: ghcr.io/php-cs-fixer/php-cs-fixer:3.41-php8.2
+  script:
+    php-cs-fixer fix --diff --dry-run --format=txt src
```

Comment:
Is it actually a correct command? Right now containers use `php-cs-fixer` as an entrypoint, so executed command should be a Fixer's command:

![image](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/30897c63-9c27-4911-bee6-aa4ef171e071)

It would be great to utilise entrypoint like [`tini`](https://github.com/krallin/tini) to make it possible to run native commands on the container too. You can see how it's done in [Composer's builds](https://github.com/composer/docker/blob/main/2.6/docker-entrypoint.sh), the difference in usage:

![image](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/adb7e981-0a48-4a7d-a1b7-113ba4e1455f)

PS. As you can see on screenshots, my platform (MacOS) is not supported natively, is it possible to make multi-arch build?

Reply:
@hgiesenow please keep in mind that I can't merge this without @keradus' decision, so if you're afraid about spending your time "for nothing", maybe wait for his comment 🙂. It can be in Q1.2024 though.

Reply:
> Is it actually a correct command? 
Yes it is, as gitlab ignores the entrypoint in favor of a shell, running the script. So this will work.

> It would be great to utilise entrypoint like [tini](https://github.com/krallin/tini) to make it possible to run native commands on the container too
I don't see the point in this yet. If you want to run a different command you can do it by overwriting the entrypoint (`docker run --entrypoint=""`). I would rather stick to docker standards then adding more "bloat" to it.

> PS. As you can see on screenshots, my platform (MacOS) is not supported natively, is it possible to make multi-arch build?
I found a documentation for that, but that would add much more complexity, so I would suggest this for a later iteration. I think it's "only" a performance topic.

Reply:
> Yes it is, as gitlab ignores the entrypoint in favor of a shell, running the script. So this will work.

👍 

> I don't see the point in this yet. If you want to run a different command you can do it by overwriting the entrypoint (docker run --entrypoint=""). I would rather stick to docker standards then adding more "bloat" to it.

Yes, but this is the worse UX, people not familiar with Docker enough will scratch their head why `docker run [...] sh` does not work 🙂. Adding `tini` keeps default action as-is, but enables user-friendly access to other commands. Not a blocker in any way, though - it can be added later.

> I found a documentation for that, but that would add much more complexity, so I would suggest this for a later iteration. I think it's "only" a performance topic.

Yes, it's a performance thing mostly and is not a blocker too 🙂.